docs/user/tutorials/01-00-x-forwarded-for-header.md

```diff
@@ -1,132 +1,82 @@
-
-Default Kyma Istio installation profile configures **PodAntiAffinity** to ensure that Ingress Gateway Pods are evenly spread across all Nodes. This guarantees that the above requirement is satisfied if your IngressGateway autoscaling configuration **minReplicas** is at least equal to the number of Nodes. You can configure autoscaling options in the Istio custom resource using **spec.config.components.ingressGateway.k8s.hpaSpec.minReplicas**.
+1. Go to **Cluster Details**. 
+2. Choose **Modify Modules**.
+2. Select the Istio module.
+3. Choose **Edit**.
+4. In the `General` section, add the number of trusted proxies.
+  Due to the variety of network topologies, the Istio CR must specify the number of trusted proxies deployed in front of the Istio Ingress Gateway proxy in the Istio CR, so that the client address can be extracted correctly.
+6. If you use a GCP or Azure custer, navigate to the Gateway section and set the Gateway traffic policy to Local. If you use a different cloud service provider, skip this step.
+   >[!WARNING]
+   > For production Deployments, it is strongly recommended to deploy Istio Ingress Gateway Pod to multiple nodes if you enable externalTrafficPolicy : Local. For more information, see Network Load Balancer.
+   >
+   >Default Istio installation profile configures PodAntiAffinity to ensure that Ingress Gateway Pods are evenly spread across all nodes. This guarantees that the above requirement is satisfied if your IngressGateway autoscaling configuration minReplicas is equal to or greater than the number of nodes. You can configure autoscaling options in the Istio CR using the field spec.config.components.ingressGateway.k8s.hpaSpec.minReplicas.
+   >[!TIP]
+   > If you use a GCP or Azure cluster, you can find your load balancer's IP address in the field status.loadBalancer.ingress of the ingress-gateway Service.
+7. Choose **Save**.
 
-> [!WARNING]
-> Deploy an Ingress Gateway Pod to multiple nodes if you enable `externalTrafficPolicy: Local` in production Deployments.
-> Switching `externalTrafficPolicy` may result in a temporal increase in request delay. Make sure that this is acceptable.
 
-> [!TIP]
-> While using GCP or Azure, you can find your load balancer's IP address in the field **status.loadBalancer.ingress** of the `ingress-gateway` Service.
-
-Add **gatewayExternalTrafficPolicy** to the Istio custom resource:
-
-<!-- tabs:start -->
 #### **kubectl**
-Run the following command:
+1. In the following command, replace the placeholder with the number of trusted proxies deployed in front of the Istio Ingress Gateway proxy. Run the command:
+  ```bash
+  kubectl patch istios/default -n kyma-system --type merge -p '{"spec":{"config":{"numTrustedProxies": NUM_OF_TRUSTED_PROXIES}}}'
+  ```
+  Due to the variety of network topologies, the Istio CR must specify the configuration property numTrustedProxies, so that the client IP address can be extracted correctly.
```

Comment:
```suggestion
  Due to the variety of network topologies, the Istio CR must specify the configuration property **numTrustedProxies**, so that the client IP address can be extracted correctly.
```